iml-gui/configs/tailwind.config.js

```diff
@@ -8,7 +8,7 @@ function* range(start, end, step) {
 // For NodeJS < 12
 Object.fromEntries =
   Object.fromEntries ||
-  (iterable => {
+  ((iterable) => {
```

Comment:
As I understand, it is the result of autoformatting?

Reply:
Yes, this is due to https://www.google.com/search?q=prettier+javascript&oq=prettier+javascript&aqs=chrome..69i57j0l7.3970j0j7&sourceid=chrome&ie=UTF-8 (basically rustfmt for js), which we are using for our js files.